evennia/commands/default/muxcommand.py

```diff
@@ -79,6 +79,11 @@ def parse(self):
         it here). The rest of the command is stored in self.args, which can
         start with the switch indicator /.
 
+        Optional variables to aid in parsing, if set:
```

Comment:
I feel `options` and `split` are a little too generic as names. `split` is what is usually done in Python when splitting by spaces; and `options` suggests (at least to me) some sort of internal option mechanic for the Command class. These are also options which will go with the other non-parser related arguments like `locks`, `key` etc. I suggest more explicit names: `parser_switch_options` and `parser_rhs_split`. 
(In comments below I'll keep referring to `options` and `split` for now)